src/bench.c

```diff
@@ -52,6 +52,7 @@
 #include "unicode.h"
 #include "config.h"
 #include "gpu_common.h"
+#include "opencl_common.h"
```

Comment:
Checking if this hurts!

Reply:
I didn't check - did this result in any warnings?

Reply:
No